integrations/nvidia.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: integration
+name: NVIDIA
+description: Use NVIDIA models with Haystack.
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/haystack-ai
+repo: https://github.com/deepset-ai/haystack
+type: Model Provider
+report_issue: https://github.com/deepset-ai/haystack/issues
+logo: /logos/nvidia.png
+version: Haystack 2.0
+toc: true
+---
+### **Table of Contents**
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+- [License](#license)
+
+## Overview
+
+ [NVIDIA AI Foundation Models](https://www.nvidia.com/en-us/ai-data-science/foundation-models/) allow you to reach optimal performance on NVIDIA accelerated infrastructure. With pretrained generative AI models, enterprises can create custom models faster and take advantage of the latest training and inference techniques. And with NVIDIA AI Foundation Endpoints, their applications can be connected to these models running on a fully accelerated stack to test performance.
+
+This integration allows you to use NVIDIA Foundation Models in your Haystack piplines.
+
+
+In order to use this integration, you'll need a [NVIDIA API key](https://org.ngc.nvidia.com/setup). Set it as an environment variable, `NVIDIA_API_KEY`.
+
+## Installation
+
+```bash
+pip install nvidia-haystack
+```
+
+## Usage
+### Components
+This integration introduces the following components:
+
+- `NvidiaTextEmbedder`: 
```

Comment:
We dont have any docs page, right?

Reply:
correct - we don't have a docs page yet.